proto/ord_schema.proto

```diff
@@ -296,74 +490,206 @@ message ReactionProduct {
-    ANALYSIS_TYPE_MP = 4;
-    ANALYSIS_TYPE_UV = 5;
-    ANALYSIS_TYPE_TLC = 6;
-    ANALYSIS_TYPE_HRMS = 7;
+    ANALYSIS_TYPE_LC = 1;  // Liquid chromatography.
+    ANALYSIS_TYPE_GC = 2;  // Gas chromatography.
+    ANALYSIS_TYPE_IR = 3;  // Infrared spectroscopy.
+    ANALYSIS_TYPE_NMR = 4;  // NMR spectroscopy.
+    ANALYSIS_TYPE_MP = 5;  // Melting point characterization.
+    ANALYSIS_TYPE_UV = 6;  // Ultraviolet spectroscopy.
+    ANALYSIS_TYPE_TLC = 7;  // Thin-layer chromatography.
+    ANALYSIS_TYPE_MS = 8;  // Mass spectrometry.
+    ANALYSIS_TYPE_HRMS = 9;  // High resolution mass spectrometry.
+    ANALYSIS_TYPE_MSMS = 10;  // Two-dimensional mass spectrometry.
+    ANALYSIS_TYPE_WEIGHT = 11;  // Weight of an isolated compound.
+    ANALYSIS_TYPE_LCMS = 12;  // Combined LC/MS.
+    ANALYSIS_TYPE_GCMS = 13;  // Combined GC/MS.
+    ANALYSIS_TYPE_ELSD = 14;  // Evaporative light scattering detector.
   }
   AnalysisType analysis_type = 1;
   // Used to describe an UNKNOWN analysis type.
-  string custom_analysis_type = 2;
-  map <string, bytes> processed_data = 3;
-  map <string, bytes> raw_data = 4;
-  string instrument_manufacturer = 5;
-  DateTime instrument_last_calibrated = 6;
+  string analysis_type_custom = 2;
+  // Any details about analysis (e.g., columns, gradients, conditions)
+  string analysis_details = 3;
+  // Data files (processed or annotated).
+  map <string, bytes> data_processed = 4;
+  // Data files (raw) obtained directly from the instrument
+  map <string, bytes> data_raw = 5;
+  string instrument_manufacturer = 6;
+  DateTime instrument_last_calibrated = 7;
 }
 
 message ReactionProvenance {
-  message Person {
-    string username = 1;
-    string name = 2;
-    string orcid = 3;
-    string organization = 4;
-  }
   Person experimenter = 1;
   string city = 2;
   DateTime experiment_start = 3;
-  oneof publication {
-    string doi = 4;
-    string patent = 5;
+  string doi = 4;
+  string patent = 5;
+  string publication_url = 6;
+  DateTime record_created = 7;
+  DateTime record_modified = 8;
+  // TODO(ccoley): is it useful to create a unique ID field 
+  // that the centralized database can write to? (I know uniqueness is not
+  // enforceable)
+  int32 id = 9;  
+}
+
+message Person {
+  string username = 1;
+  string name = 2;
+  string orcid = 3;
+  string organization = 4;
+}
+
+// TODO(ccoley) is it appropriate to have these be part of messages? It looks
+// redundant, but I think this is actually a fairly clean way to do it, rather
+// than putting all of the possible measurement/unit types into one giant enum
+/**
+ * To allow users to describe synthetic processes in whatever units they find
+ * most natural, we define a fixed list of allowable units for each measurement
+ * type. Upon submission to a centralized database, or using a validation and
+ * canonicalization script, we will convert all values to the default units
+ * (the first item in each enum).
+ */
+message Time {
```

Comment:
Maybe ReactionTime to make it clearer that we are measuring from the reaction start?

Reply:
I prefer keeping the message as just ```Time``` (since that's more like a primitive value) but specifying reaction time in fields as needed